proto/kvrpcpb.proto

```diff
@@ -711,6 +711,10 @@ message LockInfo {
     uint64 txn_size = 5;
     Op lock_type = 6;
     uint64 lock_for_update_ts = 7;
+    // Fields for transactions that are using Parallel Commit.
+    bool use_parallel_commit = 8;
```

Comment:
How about renaming it to async_commit?

Reply:
Oh, we should rename all occurances of "parallel commit" to "async commit"?

Reply:
Yes.

Reply:
Ok. Let me do this in this PR.